legal-api/report-templates/template-parts/bc-annual-report/legalObligations.html

```diff
@@ -0,0 +1,28 @@
+<div class="no-page-break">
+  <div class="section-title mt-4">Legal Obligations</div>
+
+  <div class="section-data mt-4">
+     You are required by the Business Corporations Act to keep the information about your corporation up
+      to date with the Registrar: For example, you must file annual reports, director changes and address changes.
+      The most common updates you are legally responsible to file include:
+  </div>
+  <div class="section-data mt-5">
+     <b>Annual Reports</b> - file an annual report each year within two months after
```

Comment:
These bold tags may work but not quite as expected: we have a bold font that should be used here instead.

Reply:
Fixed.